circuits/credentialAtomicQuerySigOnChainSmt.circom

```diff
@@ -0,0 +1,14 @@
+pragma circom 2.0.0;
+
+include "lib/query/credentialAtomicQuerySigOnChainSmt.circom";
+
+component main{public [userStateInOnChainSmtRoot,
+                        userNullifier,
+                        challenge,
+                        issuerID,
+                        issuerClaimNonRevState,
```

Comment:
Ideally, userStateInOnChainSmtRoot and issuerClaimNonRevState should be the same global on-chain SMT root.
Both states should be latest and can be proved on the latest root.

Reply:
No, these are different states. `userStateInOnChainSmtRoot` - the root of the on-chain SMT, which is a tree of all the states. `issuerClaimNonRevState` - personal state of the issuer. They can never be the same states.

Reply:
Doing nothing here as discussed with @demonsh